cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -120,6 +122,39 @@ def test_get_course_list(self):
         # check both course lists have same courses
         self.assertEqual(courses_list, courses_list_by_groups)
 
+    def test_get_course_list_when_ccx(self):
+        """
+        Assert that courses with CCXLocator are filter in course listing.
+        """
+        course_location = self.store.make_course_key('Org1', 'Course1', 'Run1')
+        self._create_course_with_access_groups(course_location, self.user)
+
+        # get courses through iterating all courses
+        courses_list, __ = _accessible_courses_list(self.request)
```

Comment:
- @giocalitri i think we use single ``_`` for ``gettext`` and for unused variables we sue ``__``,   
- pylintrc take ``__`` and ``_`` as good name,

 [please see](https://github.com/edx/edx-platform/blob/master/pylintrc#L103)